docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
     >>> print("ratio in recall = ", grouped_metric.ratio(method='between_groups'))    
     ratio in recall =  0.0
 
+
+Fairness metrics
+----------------
+
+Parity constraints
+^^^^^^^^^^^^^^^^^^
+In the sections below, we review the most common fairness metrics, as well
+as their underlying assumptions and suggestions for use. Each metric requires
+that some aspects of the predictor behavior be comparable across groups. In 
+the mathematical definitions below, let :math:`X` denote a feature vector 
+used for predictions, :math:`A` be a single sensitive feature (such as age 
+or race), and :math:`Y` be the true label. Parity metrics are phrased in 
+terms of expectations with respect to the distribution over :math:`(X,A,Y)`.
+
+Demographic parity
+~~~~~~~~~~~~~~~~~~
+Demographic parity is also known as *statistical parity*. A classifier
+:math:`h` satisfies demographic parity under a distribution over
+:math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair`
+
+Demographic parity can be used to assess the extent of allocation harms, as it 
+reflects an assumption that resources should be allocated proportionally 
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on assumptions: that either the dataset is not a good representation of 
+what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the world, but the world is unjust. 
+
+It's worth considering whether the assumptions underlying this measurement 
+model maintain construct validity (see :ref:`construct_validity`). 
+Consequential validity, for example, asks how the world shaped by using the 
+measurement model; in this case, the relevant question would be whether 
+demographic parity meets the criteria for establishing "fairness", itself 
+an unobservable theoretical construct, and whether satisfying demographic 
+parity brings us closer to the world we'd like to see. 
+
```

Comment:
I added the paragraph below and will add the Fair ML book as a citation if it's not in refs.bib already. Please let me know your thoughts! 

> Fairness metrics like demographic parity can also be used as optimization constraints during the machine learning model training process. However, demographic parity is not well-suited for this purpose because it does not place requirements on the exact distribution of predictions with respect to other important variables. To understand this concept further, consider an example from the Fairness in Machine Learning textbook :footcite:`fairml_book`. Suppose a company wishes to hire applicants from group A and B at the same rate. It carefully selects applicants from group A, but carelessly selects applicants from group B. In this scenario, even though members from both groups are selected at the same rate, it's likely that underqualified or unqualified applicants are chosen more frequently for one group (group B) than another. If applicants from group B consequently perform worse in comparison to applicants from group A, negative perceptions of their performance may form. If group B experiences higher levels of prejudice or discrimination, then these perceptions may reinforce existing negative stereotypes.